index.js

```diff
@@ -7,13 +7,17 @@ import Contoller, { PRIORITIES } from './src/controller';
  */
 
 // eslint-disable-next-line no-console
-let modalRef = { showModal: () => console.warn('showModal called before initialised') };
+let modalRef = { 
+  showModal: () => console.warn('showModal called before initialised'),
+};
 
 class ModalController extends React.Component {
   render() {
     return <Contoller {...this.props} ref={(ref) => { modalRef = ref; }} />;
   }
 }
 
+export const hideTopModal = (...args) => modalRef.hideTopModal(...args);
+export const hideAllModals = (...args) => modalRef.hideAllModals(...args);
```

Comment:
Not sure about this. What would be the use case for hiding the modals from out side the scope of the modal itself? We currently lack the ability to close the modal from within the modal i.e. we should pass a prop to the modal `closeModal`. Would this solve the problem this PR is trying to solve?

Reply:
I'm doing it as well. I think that if there is `showModal`, `hideModal` also should be exposed for consistency. Although, I can't think now about a valid use case